web-bundle/src/main/java/com/graphhopper/swl/StableIdEncodedValues.java

```diff
@@ -43,17 +46,39 @@ final String getStableId(boolean reverse, EdgeIteratorState edge) {
         return Long.toUnsignedString(Longs.fromByteArray(stableId));
     }
 
-    final void setStableId(boolean reverse, EdgeIteratorState edge, String stableId) {
```

Comment:
note: we weren't calling this at all before, so changing the parameter list shouldn't be an issue.